cromwell-backend/src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -0,0 +1,348 @@
+
+    val process = getCmdToExecute.run(ProcessLogger(stdoutWriter writeWithNewline, stderrTailed writeWithNewline))
+    processAbortFunc = Some(() => process.destroy())
+    subscriptions.filter(subs => subs.eventType.isInstanceOf[ExecutionEvent]).foreach(
+      subs => subs.subscriber ! new TaskStatus(Status.Running, None))
+    val backendCommandString = argv.map(s => "\"" + s + "\"").mkString(" ")
+    logger.debug(s"command: $backendCommandString")
+    val processReturnCode = process.exitValue() // blocks until process finishes
+
+    Vector(stdoutWriter.writer, stderrTailed.writer) foreach {
+      _.flushAndClose()
+    }
+
+    val stderrFileLength = Try(Files.size(stderr)).getOrElse(0L)
+
+    // TODO: check continue on error.
+
+    if (stderrFileLength > 0) {
+      TaskStatus(Status.Failed, Some(FailureResult(
+        new IllegalStateException("StdErr file is not empty."), Some(processReturnCode), stderr.toString)))
+    } else if (stderrFileLength <= 0 && processReturnCode != 0) {
+      TaskStatus(Status.Failed, Some(FailureResult(
+        new IllegalStateException("RC code is not equals to zero."), Some(processReturnCode), stderr.toString)))
+    } else {
+      def lookupFunction: String => WdlValue = WdlExpression.standardLookupFunction(task.inputs, task.declarations, expressionEval)
+      val outputsExpressions = task.outputs.map(
+        output => output.name ->(output.wdlType, output.expression.evaluate(lookupFunction, expressionEval)))
+      processOutputResult(processReturnCode, outputsExpressions)
+    }
+  }
+
+  /**
+    * Process output evaluating expressions, checking for created files and converting WdlString to WdlSimpleFile if necessary.
+    * @param processReturnCode Return code from process.
+    * @param outputsExpressions Outputs.
+    * @return TaskStatus with final status of the task.
+    */
+  private def processOutputResult(processReturnCode: Int, outputsExpressions: Seq[(String, (WdlType, Try[WdlValue]))]): TaskStatus = {
+    if (outputsExpressions.filter(_._2._2.isFailure).size > 0) {
+      TaskStatus(Status.Failed, Some(FailureResult(new IllegalStateException("Failed to evaluate output expressions.",
+        outputsExpressions.filter(_._2._2.isFailure).head._2._2.failed.get), Some(processReturnCode), stderr.toString)))
+    } else {
+      try {
+        TaskStatus(Status.Succeeded, Some(SuccesfulResult(outputsExpressions.map(
+          output => output._1 -> resolveOutputValue(output._2)).toMap)))
+      } catch {
+        case ex: Exception => TaskStatus(Status.Failed, Some(FailureResult(ex, Some(processReturnCode), stderr.toString)))
+      }
+    }
+  }
+
+  /**
+    * 1) Remove all leading newline chars
+    * 2) Remove all trailing newline AND whitespace chars
+    * 3) Remove all *leading* whitespace that's common among every line in the input string
+    *
+    * For example, the input string:
+    *
+    * "
+    * first line
+    * second line
+    * third line
+    *
+    * "
```

Comment:
Oh. This is a copy&paste of an existing functionality... I wanted to keep most of the already existing func as possible. I will remove those then.